### DISCUSSION TITLE
ColorPicker does not support RGBA, it really should.

### DISCUSSION THREAD

**Discussion by BrancuAlexandru** [primefaces/community#1198] [PrimeReact]
I'm proposing to open an issue for adding rgba() as a format option, or if that's not possible, it should be replaced with this (lightweight) library or something similar: https://www.npmjs.com/package/react-colorful

~It supports hex and rgba values~ EDIT: it support a *lot* of value types, here is the demo for rgba: https://codesandbox.io/s/react-colorful-rgb-o9q0t

**Comment by melloware** [comment]
@BrancuAlexandru there was an open ticket here: https://github.com/primefaces/primereact/issues/3937

PrimeTek said they are tracking it internally.
